README.md

```diff
@@ -129,10 +129,15 @@ an ODS index, and an NHSPD index as well as the port to run on.
 clj -M:serve /var/local/ods-2021-02  /var/local/nhspd-2020-11 8080
 ```
 
+You can now make a request to the server:
+```shell
+curl -H "Accept: application/json" "http://localhost:8080/ods/v1/search?s=hayesbank"
+```
+
 # Running a FHIR-compatible server
 
 ```shell
-clj -M:fhir-r4 /var/local/ods-2021-02 /var/local/nhspd-2020-11 8080
+clj -M:fhir_r4 /var/local/ods-2021-02 /var/local/nhspd-2020-11 8080
```

Comment:
This doesn't seem to work for me, though this change does make one error go away.

The error I'm left with is:

```
{:clojure.main/message
 "Execution error (FileNotFoundException) at java.io.FileInputStream/open0 (FileInputStream.java:-2).\ntarget/ods-2021-02 (Is a directory)\n",
 :clojure.main/triage
 {:clojure.error/class java.io.FileNotFoundException,
  :clojure.error/line -2,
  :clojure.error/cause "target/ods-2021-02 (Is a directory)",
  :clojure.error/symbol java.io.FileInputStream/open0,
  :clojure.error/source "FileInputStream.java",
  :clojure.error/phase :execution},
 :clojure.main/trace
 {:via
  [{:type java.io.FileNotFoundException,
    :message "target/ods-2021-02 (Is a directory)",
    :at [java.io.FileInputStream open0 "FileInputStream.java" -2]}],
  :trace
  [[java.io.FileInputStream open0 "FileInputStream.java" -2]
   [java.io.FileInputStream open "FileInputStream.java" 219]
   [java.io.FileInputStream <init> "FileInputStream.java" 157]
   [java.io.FileInputStream <init> "FileInputStream.java" 112]
   [clojure.lang.Compiler loadFile "Compiler.java" 7575]
   [clojure.main$load_script invokeStatic "main.clj" 475]
   [clojure.main$script_opt invokeStatic "main.clj" 535]
   [clojure.main$script_opt invoke "main.clj" 530]
   [clojure.main$main invokeStatic "main.clj" 664]
   [clojure.main$main doInvoke "main.clj" 616]
   [clojure.lang.RestFn applyTo "RestFn.java" 137]
   [clojure.lang.Var applyTo "Var.java" 705]
   [clojure.main main "main.java" 40]],
  :cause "target/ods-2021-02 (Is a directory)"}}
```

(The simple rest server _does_ work)

Reply:
Hi. Thanks for checking it out. I think the alias fhir-r4 is correct. If I use fhir_r4 I get a "specified alias is not defined" error. What platform are you running this on? You're absolutely right I need to give some examples of how to use the simple REST server. Unfortunately, the FHIR standard isn't complete enough for what I actually need - otherwise I'd provide only that API. Your error running the FHIR server is a bit odd because it has the path "target/ods-2021-02" in it and I can't see where that has come from - and it seems like it's a compilation error - so things that come to mind are: what's your working directory? what version of clojure are you using? What did you type to get that exception to happen.  For the avoidance of doubt, here's what I get using current HEAD.:

```
➜  clods git:(main) clj -M:fhir-r4 /var/tmp/ods /var/tmp/nhspd 8081
13:52:22.002 [main] INFO  org.eclipse.jetty.server.Server - jetty-9.4.18.v20190429; built: 2019-04-29T20:42:08.989Z; git: e1bc35120a6617ee3df052294e433f3a25ce7097; jvm 11.0.9.1+1
13:52:22.075 [main] INFO  com.eldrix.clods.fhir.r4.serve - Initialising HL7 FHIR R4 server; providers: Organization
(#object[com.eldrix.clods.fhir.r4.serve.OrganizationResourceProvider 0x183839f4 com.eldrix.clods.fhir.r4.serve.OrganizationResourceProvider@183839f4])
13:52:23.022 [main] INFO  org.eclipse.jetty.server.Server - Started @16415ms
```

Reply:
Ok. I've resolved the documentation issue in 87784da9ec444f83e2a9526c910ae06649ced97b but raised a separate issue #5 because I notice that the REST API doesn't expose all of the functionality of the underlying library, such as geographical proximity search.

Reply:
Worth raising a separate issue if you can't get the FHIR server working. Thanks for your help.

Reply:
Thanks @wardle, I'll see if I can get to the bottom of why the FHIR service won't run.